packages/web-components/src/dropdown/README.md

```diff
@@ -0,0 +1,176 @@
+| `end`              | often times a glyph, icon, or button follows the content                                                                            |
+| `indicator`        | holds the glyph indicating that the select can be expanded. This slot is only available if the button-container slot is not filled. |
+| `selected-value`   | displays the currently selected value text. This slot is only available if the button-container slot is not filled.                 |
+
+<br />
+<hr />
+<br />
+
+### **Suggested Template**
+
+<br />
+
+`selectTemplate` from FastFoundation
+
+<br />
+<hr />
+<br />
+
+## **Accessibility**
+
+[W3 Select Spec](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-select-element.html)
+
+<br />
+
+### **WAI-ARIA Roles, States, and Properties**
+
+<br/>
+
+**`<fluent-dropdown>`**
+
+| Attributes              | value      | Description                                                                                                                                                                          |
+| ----------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `aria-activedescendant` |            | Indicates the currently active or focused child element within a composite component.                                                                                                |
+| `aria-controls`         |            | Identifies the element(s) whose contents or presence are controlled by the current element, establishing a relationship between a controlling element and the controlled element(s). |
+| `aria-expanded`         |            | Indicates the current state of a collapsible element.                                                                                                                                |
+| `aria-haspopup`         | `listbox`  | Indicates that the current element has a popup context menu or submenu.                                                                                                              |
+| `current-value`         |            | Stores the current value of an input element. [RFC](https://github.com/microsoft/fast/issues/5119)                                                                                   |
+| `role`                  | `combobox` | Specifies the purpose or type of an element in the context of user interface components.                                                                                             |
+| `tabindex`              | `0`        | Determines the order in which an element receives focus when navigated using the keyboard.                                                                                           |
```

Comment:
These aren't exhaustive as they are missing the attributes enumerated above in "Field". I think we need to look at creating clarity in the spec given every attribute has a corresponding property but not every property may have an attribute...